core/src/server.js

```diff
@@ -11,6 +11,11 @@ const server = http.createServer(app);
 const port = config.port;
 server.listen(port, () => {
   console.log(`Soul is running on port ${port}...`);
+
+  if (config.startWithStudio) {
+    // Importing this would do the job.
+    const SoulStudio = require('soul-studio');
```

Comment:
I get this error when I try to run `npm run dev`:
```
Error: Cannot find module 'soul-studio'
Require stack:
- /home/al/dev/soul/core/src/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Server.<anonymous> (/home/al/dev/soul/core/src/server.js:18:24)
    at Object.onceWrapper (node:events:627:28)
    at Server.emit (node:events:513:28)
    at emitListeningNT (node:net:1774:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/al/dev/soul/core/src/server.js' ]
}

Node.js v18.14.2
```

Reply:
Maybe this is because the npm publish GitHub Action haven't finished its job and you already do a test.

Reply:
Please run with: `npm start -- -d :memory: -S`